bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiRpc.java

```diff
@@ -666,19 +666,21 @@ public void onNotifyEvent(Shelly2RpcNotifyEvent message) {
 
                     case SHELLY2_EVENT_OTASTART:
                         logger.debug("{}: Firmware update started: {}", thingName, getString(e.msg));
-                        getThing().postEvent(e.event, true);
-                        getThing().setThingOffline(ThingStatusDetail.FIRMWARE_UPDATING,
+                        getThing().setThingStatus(ThingStatus.UNKNOWN, ThingStatusDetail.FIRMWARE_UPDATING,
```

Comment:
According to the docs it should be offline for firmware update
https://www.openhab.org/docs/concepts/things.html#status-details

Reply:
changed